v8-riscv-tools/init_cross_env.sh

```diff
@@ -0,0 +1,140 @@
+#!/bin/bash
+
+# Note: This script is used for the V8 RISCV Developers.
+#       You may need to modify some folders to fit your need.
+
+set -e
+
+# Config: modify it if you don't like the default path.
+# IMPORTANT: please assert there is no space in pwd folder, or the script may
+#            does undefined behaviors.
+V8_ROOT=$PWD
+RV_HOME=$HOME/opt/riscv
+
+# NOTE: This script supposes you could build a sim build successfully.
+#       which means the depot_tools and other prebuilt binary blobs v8 need were ready.
+
+echo "Now we are going to download and build RISC-V GNU toolchain and QEMU."
+echo "A few repos and files will be downloaded at $PWD,"
+echo "And the toolchain is going to be installed at $RV_HOME ."
+echo "Press Ctrl-C if you do not like these configs, otherwise I will continue:"
+read -p "You have 10s to cancel:" -t 10
+
+# +++++++++++++++++++++++++++++++++++++++++
+# RISC-V GNU Toolchain
+# +++++++++++++++++++++++++++++++++++++++++
+
+mkdir -p $RV_HOME
+
+cd $V8_ROOT/
+git clone https://github.com/riscv/riscv-gnu-toolchain
+pushd riscv-gnu-toolchain
+git submodule update --init --recursive
+./configure --prefix=$RV_HOME
+make linux -j $(nproc) || make linux -j 1
```

Comment:
Is `nproc` set in the environment?  If so, it is better to allow users to specify `nproc` on command line.

Reply:
`nproc` is an util provided by coreutils. it returns the num of cores the host has.

Reply:
indeed. it is good to let the user set the parallel number. I'll update it.